weave/trace/settings.py

```diff
@@ -128,6 +128,14 @@ class UserSettings(BaseModel):
     ~/.cache/wandb/weave-scorers.
 
     Can be overridden with the environment variable `WEAVE_SCORERS_DIR`
+
+    """
+    max_calls_queue_size: int = 100_000
+    """
+    Sets the maximum size of the calls queue.  Defaults to 100_000.
+    Setting a value of 0 means the queue can grow unbounded.
+
+    Can be overridden with the environment variable `WEAVE_MAX_CALLS_QUEUE_SIZE`
```

Comment:
Could you update `docs/docs/guides/core-types/env-vars.md`

Reply:
yep, done